private/gen/proto/go/buf/alpha/audit/v1alpha1/event.pb.go

```diff
@@ -0,0 +1,576 @@
+// Copyright 2020-2022 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Code generated by protoc-gen-go. DO NOT EDIT.
+// versions:
+// 	protoc-gen-go v1.28.0
+// 	protoc        (unknown)
+// source: buf/alpha/audit/v1alpha1/event.proto
+
+package auditv1alpha1
+
+import (
+	protoreflect "google.golang.org/protobuf/reflect/protoreflect"
+	protoimpl "google.golang.org/protobuf/runtime/protoimpl"
+	timestamppb "google.golang.org/protobuf/types/known/timestamppb"
+	reflect "reflect"
+	sync "sync"
+)
+
+const (
+	// Verify that this generated code is sufficiently up-to-date.
+	_ = protoimpl.EnforceVersion(20 - protoimpl.MinVersion)
+	// Verify that runtime/protoimpl is sufficiently up-to-date.
+	_ = protoimpl.EnforceVersion(protoimpl.MaxVersion - 20)
+)
+
+// ActorType is the type of actor that caused the audited event.
+type ActorType int32
+
+const (
+	ActorType_ACTOR_TYPE_UNSPECIFIED ActorType = 0
```

Comment:
Looks like the `ACTOR_TYPE_` prefix in `proto/buf/alpha/audit/v1alpha1/event.proto` is creating a stutter here. Is this a common convention, or should we drop it?

Reply:
ye, it's just an unfortunate fallout of:

> Enum value names should be prefixed with the UPPER_SNAKE_CASE of the enum name. [Why?](https://docs.buf.build/lint/rules#enum_value_prefix)

https://docs.buf.build/best-practices/style-guide#enums

Reply:
ah, cool, okay 🙃 

thanks